schemas/threat.yml

```diff
@@ -0,0 +1,85 @@
+---
+- name: threat
+  title: Threat
+  group: 2
+  short: Fields to classify events and alerts according to a threat taxonomy.
+  description: >
+   Fields to classify events and alerts according to a threat taxonomy such as the Mitre ATT&CK framework.
+
+   These fields are for users to classify alerts from all of their sources (e.g. IDS, NGFW, etc.) within a 
+   common taxonomy. The threat.technique.* are meant to capture the high level category of the threat 
+   (e.g. "exfiltration"). The threat.tactic.* fields are meant to capture which kind of approach is used by 
+   this detected threat, to accomplish the goal (e.g. "data compressed").
```

Comment:
I asked you to adjust the fieldset definition, and I got tactic and technique reversed 😂 Sorry about that.

High level category is tactic, and technique is the actual method used. Please update it to undo my mistake:

> These fields are for users to classify alerts from all of their sources (e.g. IDS, NGFW, etc.) within a 
   common taxonomy. The threat.tactic.* fields are meant to capture the high level category of the threat 
   (e.g. "exfiltration"). The threat.technique.* fields are meant to capture the method used by 
   this detected threat, to accomplish the goal (e.g. "data compressed").